### PR TITLE
[TRUS-4611] - Other Beneficiaries - Registration

### PR DESCRIPTION
**Pull request by jassalrichy**
Update Non-Taxable Navigation

### CODE REVIEW

File: app/navigation/OtherBeneficiaryNavigator.scala [hunk 1]
```diff
@@ -34,11 +34,23 @@ class OtherBeneficiaryNavigator @Inject()() extends Navigator {
       yesNoNavigation(draftId)
 
   private def simpleNavigation(draftId: String):  PartialFunction[Page, ReadableUserAnswers => Call] = {
-    case DescriptionPage(index) => _ => other.routes.DiscretionYesNoController.onPageLoad(index, draftId)
+//    case DescriptionPage(index) => _ => other.routes.DiscretionYesNoController.onPageLoad(index, draftId)
```
Comment (BriWak) [inline app/navigation/OtherBeneficiaryNavigator.scala:37]: can we get rid of the commented out code or has it been left there for a reason?

File: app/navigation/OtherBeneficiaryNavigator.scala [hunk 2]
```diff
@@ -34,11 +34,23 @@ class OtherBeneficiaryNavigator @Inject()() extends Navigator {
...
+      if (is5mldNonTaxable(ua)) {
+        other.mld5.routes.CountryOfResidenceYesNoController.onPageLoad(index, draftId)
+      } else {
+        other.routes.DiscretionYesNoController.onPageLoad(index, draftId)
+      }
     case ShareOfIncomePage(index) => ua => navigateAwayFromIncomeQuestions(draftId, index, ua.is5mldEnabled)
     case AddressUKPage(index) => _ => other.routes.CheckDetailsController.onPageLoad(index, draftId)
     case AddressInternationalPage(index) =>_ =>  other.routes.CheckDetailsController.onPageLoad(index, draftId)
-    case CountryOfResidencePage(index) => _ => other.routes.AddressYesNoController.onPageLoad(index, draftId)
+//    case CountryOfResidencePage(index) => _ => other.routes.AddressYesNoController.onPageLoad(index, draftId)
```
Comment (BriWak) [inline app/navigation/OtherBeneficiaryNavigator.scala:47]: and here

**Review by BriWak — CHANGES_REQUESTED** [review]

File: app/navigation/OtherBeneficiaryNavigator.scala (hunk 1 above)
Comment (jassalrichy) [reply app/navigation/OtherBeneficiaryNavigator.scala]: Updated

File: app/navigation/OtherBeneficiaryNavigator.scala (hunk 2 above)
Comment (jassalrichy) [reply app/navigation/OtherBeneficiaryNavigator.scala]: Updated

**Review by BriWak — APPROVED** [review]

**Pull request merged**